README.md

```diff
@@ -1,79 +1,97 @@
 # Overview
-[LINE API Use Caseサイト](https://lineapiusecase.com/ja/top.html)で提供している[ヘアサロン予約](https://lineapiusecase.com/ja/usecase/reservation.html)のデモアプリケーションソースコードとなります。    
-今回紹介している手順を参考にすると、LINE APIを活用したヘアサロン予約アプリケーションを開発することが可能です。    
-ヘアサロン予約アプリケーションはLINEアプリでLIFFブラウザ上で起動し、ヘアサロンの予約を行うことが出来ます。  
-ユーザーの来店忘れを防止するための、LINEメッセージによるリマインド機能もデフォルトで実装できます。  
 
-なお、このページで紹介しているソースコードの環境はAWSを利用しています。  
-※ ドキュメントなどの文言は日本語対応となっています。  
-※ This document is written in only Japanese for now. We’ll translate it later as soon as possible.
+This is the demo app source code for the [Hair Salon Reservation](https://lineapiusecase.com/en/usecase/reservation.html) provided on the [LINE API Use Case site](https://lineapiusecase.com/en/top.html). By referring to the steps described in this article, you can develop a hair salon reservation app using the LINE API. The hair salon reservation app is launched on the LIFF browser of the LINE app, and you can make a reservation at the hair salon. To prevent users from forgetting to visit the store, you can implement a reminder function via LINE message by default.
+
+The source code environment introduced on this page uses AWS.
+
+*Documentation and other content are only available in Japanese.
 
 # Libraries
+
 ## Node.js
-フロントエンド側の開発で使用する Node.js をローカル開発環境にインストールしてください。  
-※ v10.13 以上 最新の LTS バージョンのインストールをおすすめします
 
-【Node.jsダウンロードサイト】  
-https://nodejs.org/ja/download/
+Install Node.js, which will be used for the front-end development, in your local development environment.
+
+*We recommend installing the latest Long-Term Support (LTS) version (v10.13 or later).
+
+- [Download Node.js](https://nodejs.org/ja/download/)
 
 ## Python
-Pythonのバージョン3.8以上がインストール済みでない場合、インストールしてください。  
-コマンドプロンプト、又はターミナルにて以下のコマンドを入力し、インストール済みか確認できます。
+
+Install Python version 3.8 or later if it isn't already installed.
+
+You can check if it's installed by entering this command in Command Prompt or Terminal:
+
 ```
 python --version
+```
 
-Python 3.8.3 ← このように表示されたら、インストール済みです。
+If Python is installed, the version will be returned. For example:
+
+```
+Python 3.8.3
 ```
 
-インストール済みでない場合、バックエンド側の開発で使用するPython（3.8以上）をローカル開発環境にインストールしてください。
+Install Python (3.8 or higher), used for backend development, in your local development environment if it's not already installed.
```

Comment:
In the [previous sentence](https://github.com/line/line-api-use-case-reservation-hairsalon/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R13), `front-end` is used, and here `backend` without a hyphen is used. 
I think both are completely fine, but could you make the syntax consistence?

Reply:
Sure can

Reply:
```suggestion
Install Python (3.8 or higher), used for back-end development, in your local development environment if it's not already installed.
```